src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -885,7 +885,7 @@ export class CalciteCombobox {
     const single = this.selectionMode === "single";
     const labelId = `${guid}-label`;
     return (
-      <Host>
+      <Fragment>
```

Comment:
Should we keep `Host` when there's more than one child? Using `Fragment` doesn't reduce nesting and I'm not sure what we gain here.

Reply:
Yeah, I guess we just protect from modifying the host.

Reply:
Cool. We've got the new ESLint rules for that too, so I think the combo will help keep the host clean.